compiler/src/dotty/tools/dotc/config/Settings.scala

```diff
@@ -106,14 +106,15 @@ object Settings:
     preferPrevious: Boolean = false,
     propertyClass: Option[Class[?]] = None,
     deprecation: Option[Deprecation] = None,
+    deprecatedAliases: List[(String, Deprecation)] = Nil,
```

Comment:
nitpick: wouldn't it increase readability to have a 
```
case class SettingAlias(name: String, deprecation: Option[Deprecation] = None)
```
?
or at the very least, use a named tuple here 🤔